apps/frontend/src/components/template/FapReviewTemplatesTable.tsx

```diff
@@ -0,0 +1,208 @@
+import MaterialTable, { Column } from '@material-table/core';
+import Button from '@mui/material/Button';
+import Link from '@mui/material/Link';
+import React, { useCallback, useState } from 'react';
+import { Link as ReactRouterLink } from 'react-router-dom';
+
+import { ActionButtonContainer } from 'components/common/ActionButtonContainer';
+import InputDialog from 'components/common/InputDialog';
+import { FapReviewTemplate, Review, TemplateGroupId } from 'generated/sdk';
+import { useFormattedDateTime } from 'hooks/admin/useFormattedDateTime';
+import { useCallsData } from 'hooks/call/useCallsData';
+import { useReviewsData } from 'hooks/review/useReviewsData';
+import { tableIcons } from 'utils/materialIcons';
+
+import TemplatesTable, { TemplateRowDataType } from './TemplatesTable';
+
+function CallsList(props: { filterTemplateId: number }) {
+  const { calls } = useCallsData({ templateIds: [props.filterTemplateId] });
+  const { toFormattedDateTime, timezone } = useFormattedDateTime({
+    shouldUseTimeZone: true,
+  });
+
+  const callListColumns = [
+    { title: 'Short Code', field: 'shortCode' },
+    {
+      title: `Start Date(${timezone})`,
+      field: 'startCallFormatted',
+    },
+    {
+      title: `End Date(${timezone})`,
+      field: 'endCallFormatted',
+    },
+  ];
+
+  const callsWithFormattedDates = calls.map((call) => ({
+    ...call,
+    startCallFormatted: toFormattedDateTime(call.startCall),
+    endCallFormatted: toFormattedDateTime(call.endCall),
+  }));
+
+  return (
+    <MaterialTable
```

Comment:
Isn't it better to use the loading state on the table? For example if you extract `loading` state from calls and add it to the `MaterialTable` `isLoading` property.

Reply:
Done